build/ci.go

```diff
@@ -149,10 +144,6 @@ var (
 	// versions.
 	gobootVersion = "1.19.6"
```

Comment:
You forgot to also use the version for this from the checksums. Or was i deliberate?

Reply:
It was not deliberate, no

Reply:
Fixed, ptal